plugins/jira/komand_jira/triggers/updated_issue/trigger.py

```diff
@@ -0,0 +1,66 @@
+import insightconnect_plugin_runtime
+import time
+from .schema import UpdatedIssueInput, UpdatedIssueOutput, Input, Output, Component
+
+# Custom imports below
+from komand_jira.util.util import normalize_issue, look_up_project
+from insightconnect_plugin_runtime.exceptions import PluginException
+
+
+class UpdatedIssue(insightconnect_plugin_runtime.Trigger):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="updated_issue",
+            description=Component.DESCRIPTION,
+            input=UpdatedIssueInput(),
+            output=UpdatedIssueOutput(),
+        )
+        self.get_attachments = None
+        self.projects = []
+        self.jql = ""
+
+    def poll(self):
+        new_issues = self.connection.client.search_issues(self.jql, startAt=0, maxResults=False, fields="*all")
+        for issue in new_issues:
+            if issue.id:
+                output = normalize_issue(issue, get_attachments=self.get_attachments, logger=self.logger)
+                self.logger.debug(f"Found: {output}")
+                self.send({Output.ISSUE: output})
+
+    def validate_projects(self, projects):
+        for project in projects:
+            valid_project = look_up_project(project, self.connection.client)
+            if not valid_project:
+                raise PluginException(
+                    cause=f"Project '{project}' does not exist or the user does not have permission to access the "
+                    "project.",
+                    assistance="Please provide a valid project ID/name or make sure the project is accessible to the "
+                    "user.",
+                )
+
+    def run(self, params={}):
+        """Run the trigger"""
+        self.jql = params.get(Input.JQL)
+        self.projects = params.get(Input.PROJECTS)
+        self.get_attachments = params.get(Input.GET_ATTACHMENTS, False)
+
+        jql = "(created>=-10m or updated>=-10m)"
```

Comment:
The ticket around this stated they were using last 20 minutes - is there any reason we are going with last 10?

Reply:
Hi @mrinehart-r7! I think it would be a good idea to use the user-specified `interval` instead of 10 or 20 minutes, this way we avoid sending the same information repeatedly by the `poll` method. What do you think about it?

Reply:
> Hi @mrinehart-r7! I think it would be a good idea to use the user-specified `interval` instead of 10 or 20 minutes, this way we avoid sending the same information repeatedly by the `poll` method. What do you think about it?

Hi @dsliwinski-r7 ! I think user-defined is best, but the ticket stated that it was causing a defect by polling too soon, which is why they suggested 20 minutes.

Reply:
@mrinehart-r7 I changed it to the suggested 20 minutes and added a method that prevents multiple sending of data with the same issue :)